sdk/ai/azure-ai-generative/pyproject.toml

```diff
@@ -7,5 +7,5 @@ pylint = false
 black = false
 strict_sphinx = false
```

Comment:
let's remove `strict_sphinx` since that also controls if docs are built

Reply:
I'm not 100% sure `strict_sphinx = false` is different from `strict_sphinx` being missing, but either way resolved by https://github.com/Azure/azure-sdk-for-python/pull/32884/commits/5e0612fee65d65905acefb3a8a4ff43e297532a0